### PR TITLE
Planet sandbox data landing page

### PR DESCRIPTION
**Pull request by zcernigoj**
Landing page for https://collections.sentinel-hub.com/planet-sandbox-data/

- all sandbox related files will be named `sandbox_[something]`, so I renamed `about-sandbox.hbs` to `sandbox_about.hbs`
- added partial page for creating collections list (`_build/partials/sandbox_collectionsList.hbs`)
- added styling for the collections list on sandbox data landing page (`_build/css/main.css`)
- added file that combines partial pages (`sandbox_about.hbs`, `sandbox_collectionsList.hbs`) to a whole page (`_build/sandbox_index.hbs`)
- added function to `gulpfile.js` to generate the html code from the `sandbox_index.hbs` file for sandbox data landing page

### CODE REVIEW

File: _build/css/main.css [hunk 1]
```diff
@@ -373,4 +373,26 @@ table>tbody+tbody{
...
+}
+
+.sandbox-dataset .sandbox-dataset-title {
+  color: black;
+}
+
+.sandbox-dataset .image-card {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
```
Comment (dthiex) [inline _build/css/main.css:385]: Was this somewhere discussed? In the design images are aligned "start".

File: _build/partials/sandbox_about.hbs
Comment (dthiex) [inline _build/partials/sandbox_about.hbs:1]: Still missing:
- Planet logo
- ~css for links within text~ (I think links are actually already okay but there is one missing for the ARPS data).

File: _build/partials/sandbox_collectionsList.hbs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            </a>
+          </div>
+        {{/if}}
+        <div class="info">
+          <p>{{{trimHTML (md Description) 500}}}</p>
+          <p><a href="{{rootUrl}}{{Slug}}/sandbox-data.html">Sandbox data for {{Name}} &rarr;</a></p>
+        </div>
+
+      </div>
+
```
Comment (dthiex) [inline _build/partials/sandbox_collectionsList.hbs:21]: Check and remove empty lines where not needed ;)

**Review by dthiex — CHANGES_REQUESTED** [review]
Somewhat connected, can you add a bit more space between the tags when they break into two lines? I currently looks very cramped. 

grafik

**Comment by zcernigoj** [comment]
This is needed as soon as possible, so merging.

Will put the fixes in a separate MR

**Comment by zcernigoj** [comment]
> Somewhat connected, can you add a bit more space between the tags when they break into two lines? I currently looks very cramped.
> 
> grafik

should be fixed in https://github.com/sentinel-hub/public-collections/pull/114

File: _build/partials/sandbox_collectionsList.hbs (hunk 2 above)
Comment (zcernigoj) [reply _build/partials/sandbox_collectionsList.hbs:21]: should be fixed in https://github.com/sentinel-hub/public-collections/pull/114

File: _build/partials/sandbox_about.hbs
Comment (zcernigoj) [reply _build/partials/sandbox_about.hbs:1]: should be fixed in https://github.com/sentinel-hub/public-collections/pull/114

File: _build/css/main.css (hunk 1 above)
Comment (zcernigoj) [reply _build/css/main.css:385]: should be fixed in https://github.com/sentinel-hub/public-collections/pull/114

**Pull request closed without merging**